spec/rubocop/cop/rspec/missing_example_group_argument_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe RuboCop::Cop::RSpec::MissingExampleGroupArgument do
+  subject(:cop) { described_class.new }
+
+  it 'accepts describe with an argument' do
+    expect_no_offenses(<<-RUBY)
+      describe FooClass do
+      end
+
+      RSpec.describe FooClass do
+      end
+    RUBY
+  end
+
+  it 'accepts methods with a name like an example block' do
+    expect_no_offenses(<<-RUBY)
+      describe FooClass do
+        def describe
```

Comment:
This is not a good false-positive test, as the node is of `def` type.
Use something like Scenario.context { "static" }, e.g. you need a method call

Reply:
Sorry - got it wrong. Hopefully its now ok like this.